tests/integration/wifi.py

```diff
@@ -152,7 +152,7 @@ class TestNetworkManager(IntegrationTestsWifi, _CommonTests):
 
     def test_wifi_ap_open(self):
         # we use dev_w_client and dev_w_ap in switched roles here, to keep the
-        # existing device blacklisting in NM; i. e. dev_w_client is the
+        # existing device blocklisting in NM; i. e. dev_w_client is the
```

Comment:
nitpick: I'd prefer `denylisting` here, "blocklisting" makes me think about a block of devices.